pkg/backends/calico/routes_suite_test.go

```diff
@@ -0,0 +1,18 @@
+package calico
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/onsi/ginkgo/reporters"
+	"github.com/projectcalico/libcalico-go/lib/testutils"
+)
+
+func TestCalico(t *testing.T) {
+	testutils.HookLogrusForGinkgo()
+	RegisterFailHandler(Fail)
+	junitReporter := reporters.NewJUnitReporter("../../../report/calico_backend_suite.xml")
```

Comment:
Other repos have a `report/README.txt` file checked in to ensure that the `report` dir exists.  I wonder if we need that here too.

Reply:
Yes we do; the run says:
```
Random Seed: 1542885195
Will run 8 of 8 specs

••••••••Failed to create JUnit report file: ../../../report/calico_backend_suite.xml
	open ../../../report/calico_backend_suite.xml: no such file or directoryFailed to generate JUnit report
	invalid argument
```